src/sentry/models/distribution.py

```diff
@@ -0,0 +1,57 @@
+"""
+sentry.models.distribution
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2017 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+from django.db import models, IntegrityError, transaction
+from django.utils import timezone
+
+from sentry.db.models import Model, FlexibleForeignKey, sane_repr
+
+
+class Distribution(Model):
+    __core__ = False
+
+    release = FlexibleForeignKey('sentry.Release')
+    name = models.CharField(max_length=64)
+    date_added = models.DateTimeField()
+
+    class Meta:
+        app_label = 'sentry'
+        db_table = 'sentry_distribution'
+        unique_together = (('release', 'name'),)
+
+    __repr__ = sane_repr('release', 'name')
+
+    @classmethod
+    def get_or_create(cls, release, name, date_added=None):
```

Comment:
I think vastly is an overstatement compared to behavior not even being correct. "get or create" is wrong, when the behavior is "create or get".

If you're insistent on this behavior, can we just have like:

```python
def get_or_create(...):
  return Distribution.objects.get_or_create(...)[0]
```

just so the name matches what it does, with a comment explaining why it's here.